vortex-proto/Cargo.toml

```diff
@@ -22,5 +22,5 @@ dtype = []
 scalar = ["dtype"]
 expr = ["dtype", "scalar"]
 
-[lints]
-workspace = true
+#[lints]
+#workspace = true
```

Comment:
I couldn't figure out a better way to kill the clippy errors on the generated code.

I tried both `#[allow(clippy)]` and `#[allow(clippy::all)]` on the module declarations and that didn't do it 🤷

Reply:
Clippy::all doesn’t mean “all” it means the things that are not pedantic or nursery or something like that